.golangci.yml

```diff
@@ -125,41 +126,30 @@ issues:
       linters:
         - govet
 
-    # Exclude known linters from partially hard-vendored code,
-    # which is impossible to exclude via "nolint" comments.
-    - path: internal/hmac/
-      text: "weak cryptographic primitive"
+    - text: "`finalized` is a misspelling of `finalised`"
```

Comment:
does this ignore this? or what does this mean

Reply:
yes that instructs it to ignore that misspelling error, since we have a bunch of `finalized` words (I guess due to external spec in american english and not UK english).